packages/common/src/Environments.ts

```diff
@@ -13,6 +13,7 @@ export interface Environment {
   readonly gtxdatazero: number
   readonly dataOnChainHandlingGasCostPerByte: number
   readonly getGasPriceFactor: number
+  readonly nonZeroDevOverhead: number
```

Comment:
First, I assume this is the gas cost of transferring the dev fee? So it is only added when the fee is "not zero"? 
I would not name it like that. We have "gtxdatanonzero" and it is related to a calldata byte and this comes directly from the Yellow Paper.

How about "devFeeTransferringOverhead"?


Second, I don't see it being used outside of tests. Maybe we shouldn't add an Environment variable that is not used in code?

Reply:
It should be used to set new gasOverhead once we define dev fee.

Reply:
1. I asked two questions. 
2. I don't understand your answer.

Reply:
1. I think the name is fine
2. Read again